cmd/create/cluster/cmd.go

```diff
@@ -250,6 +255,20 @@ func init() {
 			"Leave empty to use an auto-generated one.",
 	)
 
+	flags.StringVar(
+		&args.oidcEndpointUrl,
+		"oidc-endpoint-url",
+		"",
+		"Endpoint url for BYO OIDC config",
+	)
```

Comment:
You should add `MarkHidden` to these new flags for now.

Reply:
Sure